experimental/entrypoint.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+echo "Running pycodestyle."
+pycodestyle --exclude=vendor --ignore=E402,W503,E501 target_finder_model generate scripts_tf test
+
+echo "Building data."
+python scripts_generate/build.py
+
+echo "Creating TF-Records."
+python scripts_tf/create_tf_records.py
+
+cp -r model_data/records /host/model_data
+
+echo "Creating label map."
+python scripts_tf/create_label_map.py
+
+echo "Training classification model."
+#python scripts_tf/train_clf.py
+# This command will fail on CI builds without gpu ()
+if [ $? -eq 0 ]; then
```

Comment:
It might be better to do an `if isCI...else...` rather than ignoring failed output.

Reply:
Yeah this script could use some work. I am going to leave it in experimental since its WIP, but maybe could be tweaked for future use.